tests/test_black.py

```diff
@@ -31,7 +31,7 @@ def test_black_works(
     # check diff
     with open(tmp_notebook_for_testing) as handle:
         before = handle.readlines()
-    path = os.path.abspath(os.path.join("tests", "data", "notebook_for_testing.ipynb"))
+    path = os.path.join("tests", "data", "notebook_for_testing.ipynb")
```

Comment:
Why not define a variable `TEST_DATA_PATH` at the root of tests and use this instead?
`TEST_DATA_PATH` could be of type `pathlib.Path` since it is only used in a strings, so no `str(path)` overhead.
That way this line (and all the similar ones) would be:
```python
path = TEST_DATA_PATH / "notebook_for_testing.ipynb"
```
Which IMHO is much more readable.

Reply:
100% agreed :+1: I'll do some much-needed cleaning up / refactoring here in a separate PR (I prefer to not mix changing logic with refactoring)